app/models/edition.rb

```diff
@@ -424,6 +425,22 @@ def can_be_tagged_to_taxonomy?
     false
   end
 
+  def expanded_links
```

Comment:
Would it be possible to make a single call to `publishing_api.get_links()` to see if the item has been tagged? Then we don't need the gem & the `LinkedContentItem` setup.